package.json

```diff
@@ -20,7 +20,9 @@
   },
   "homepage": "https://github.com/internetarchive/bookreader#readme",
   "dependencies": {
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "lodash": "^4.17.15",
+    "sinon": "^7.3.2"
```

Comment:
Can `lodash` and `sinon` go in the `devDependencies` below?

Reply:
Yes, i'll do it